src/styles/components/navbar/_navbar.scss

```diff
@@ -15,24 +15,44 @@ header {
 
```

Comment:
Can you remove `header {width: 100vw }` a few lines above. It adds a horizontal scrollbar when a vertical appears at some sizes.

Reply:
done